httpserver/lmq_server.cpp

```diff
@@ -265,4 +305,39 @@ void OxenmqServer::init(ServiceNode* sn, RequestHandler* rh, oxenmq::address oxe
     connect_oxend(oxend_rpc);
 }
 
+std::string OxenmqServer::encode_onion_data(std::string_view payload, const OnionRequestMetadata& data) {
+    return oxenmq::bt_serialize<oxenmq::bt_dict>({
+            {"d", payload},

```

Comment:
probably unnecessary, but better to have longer / more descriptive dict keys

Reply:
Longer is better.